bin/disco_ssh.py

```diff
@@ -133,16 +135,31 @@ def build_ssh_cmd(self, ips):
         """
         Given a list of ip addresses, build an ssh command to tunnel through n-1 ips to reach the n-th ip
         """
-        command = " ".join([
-            "ssh -At {} {}".format(SSH_OPTIONS, ip)
-            for ip in ips])
+        if self.tunnel:
+            lport, rport = self.tunnel.split(":")
+            host = self.args["<host>"]
```

Comment:
This seems like a possible cause of ambiguity—is the "host" argument required to be an addressable domain name if `--tunnel` is used?